BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -98,6 +103,9 @@ private void updateLocation() {
     }
 
     private void onLocationUpdated(Location location) {
+        if (maximumMovementSpeed != MAXIMUM_MOVEMENT_SPEED_NOT_SET && lastKnownLocation != null) {
+            location = DistanceUtil.speedFilter(lastKnownLocation, location, maximumMovementSpeed);
```

Comment:
Why is a `speedFilter` useful here?

Reply:
To adjust the location, if a maximum speed is set